.github/workflows/interactions-publish-to-npm.yml

```
@@ -0,0 +1,42 @@
+name: Wasm publish to npm
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to publish Wasm under (e.g. e.g. `dev`, will default to `latest`)'
+        required: true
+      branch:
+        description: 'Branch to run publish from'
+        required: true
+      dry-run:
+        description: 'Run in dry-run mode'
+        type: boolean
+        required: false
+        default: true
+
+jobs:
+
+  build-wasm: 
+    uses: './.github/workflows/shared-build-wasm.yml'
+    with:
+      run-unit-tests: false
+      ref: ${{ github.event.inputs.branch }}
+      output-artifact-name: identity-wasm-bindings-build
+
+  release-wasm:
+    runs-on: ubuntu-latest
+    needs: [build-wasm]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.branch }}
+      - name: Release to npm
+        uses: './.github/actions/publish/publish-wasm'
+        with:
+          dry-run: ${{ github.event.inputs.dry-run }}
+          input-artifact-name: identity-wasm-bindings-build
```

Should the artifact name here be different? E.g. something like

```suggestion
          input-artifact-name: iota-interaction-bindings-build
```

---

it's the same artifact containing both ATM